client.go

```diff
@@ -12,14 +12,13 @@ import (
 	"sync"
 	"time"
 
-	"github.com/google/uuid"
 	ipfslite "github.com/hsanjuan/ipfs-lite"
 	"github.com/ipfs/go-cid"
 	"github.com/mr-tron/base58"
 	ma "github.com/multiformats/go-multiaddr"
 	"github.com/textileio/go-foldersync/watcher"
-	core "github.com/textileio/go-textile-core/store"
-	es "github.com/textileio/go-threads/eventstore"
+	models "github.com/textileio/go-threads/core/store"
+	core "github.com/textileio/go-threads/store"
```

Comment:
Little confusing that the core module is aliased to `models`, and the store module to `core`. I've been not aliasing go-threads/store, and calling the core variant `core`.

Reply:
ah yep, moved to quickly there. fixed